modules/gltf/structures/gltf_node.h

```diff
@@ -53,6 +53,7 @@ class GLTFNode : public Resource {
 	Vector3 scale = Vector3(1, 1, 1);
 	Vector<int> children;
 	GLTFLightIndex light = -1;
+	Dictionary additional_data;
 
```

Comment:
I still have some misgivings about how this works. Maybe we allocate a dict for each extension by name rather than sharing one dictionary. Thoughts?

Reply:
How would you suggest this be done?